common/djangoapps/util/course.py

```diff
@@ -76,12 +76,11 @@ def has_certificates_enabled(course):
 def should_display_grade(course_overview):
     """
     Returns True or False depending upon either certificate available date
-    or course-end-date
+    or course end date
     """
-    course_end_date = course_overview.end_date
     cert_available_date = course_overview.certificate_available_date
     current_date = now().replace(hour=0, minute=0, second=0, microsecond=0)
     if cert_available_date:
         return cert_available_date < current_date
 
-    return course_end_date and course_end_date < current_date
+    return course_overview.end and course_overview.end < current_date
```

Comment:
could you explain the difference between course_overview.end_date and course_overview.end?

Reply:
Yep, both are the end date. We believed we needed to not use protected keywords (such as start and end) at one point so DE began the process of switching from start/end to start_date/end_date. Then they discovered that due to a different setting in our integration, start/end would actually function just fine and they decided to stop the work of the rename. This PR just sets the preferred value back to start and end and discourages use of start_date/end_date until actual database cleanup can be dealt with.